gcloud/orb.yaml

```diff
@@ -483,5 +483,56 @@ jobs:
           image: <<parameters.project>>/<<parameters.image>>
           registry: <<parameters.registry>>
 
+  artifacts-package-publish:
+    description: >
+      This job will publish an already built package to a GCP package
+      repository.
+    docker:
+      - image: <<parameters.executor>>
+    resource_class: <<parameters.resource_class>>
+    parameters:
+      creds:
+        default: GCLOUD_SERVICE_KEY
+        description: >
+          Name of environment variable storing the base64-encoded service key
+          for the GCP project.
+        type: env_var_name
+      executor:
+        default: python:3.7.10
+        description: >
+          Name of the python image to use to execute the job.
+        type: string
+      path:
+        default: .
+        description: >
+          Path to the directory containing your packages.
+        type: string
+      project:
+        description: >
+          Name of GCP project to which we will push.
+        type: string
+      resource_class:
+        default: medium
+        type: string
+      repository-url:
+        description: >
+          URL of GCP package repository to which we will push.
+        type: string
+      workspace:
+        default: '/tmp/build_dir'
+        description: >
+          If specified, attaches the specified workspace and copies it into the
+          image context before building.
+        type: string
+    steps:
+      - run: pip install twine keyring keyrings.google-artifactregistry-auth
+      - install
+      - auth:
+          creds: <<parameters.creds>>
+          project: <<parameters.project>>
+      - attach_workspace:
+            at: <<parameters.workspace>>
+      - run: twine upload --repository-url <<parameters.repository-url>> <<parameters.workspace>>/<<parameters.path>>/dist/*
```

Comment:
This line should go into the docs, eg. in the top-level `description` something along the lines of "your CI workspace must contain the packages you want to deploy in a subfolder named `<<parameters.path>>/dist/`". To keep things simpler, you might actually want to do away with the `dist` part entirely and just define `path` as "the subfolder within your workspace containing the packages to be uploaded"